cli/verbs.go

```diff
@@ -126,6 +126,17 @@ func run(requestor string, transport transport.Transport, stdout io.Writer) erro
 				filename = env.DefaultLogFile
 			}
 
+			_, err = os.Stat(filename)
+			if err != nil {
+				if errors.Is(err, os.ErrNotExist) {
+					if err := os.MkdirAll(filename, 0666); err != nil {
```

Comment:
I think we need to separate the base folder from the filename und use the basepath in the call to MkdirAll. Otherwise the filename will be the last folder to be created, which might cause things to  break when we create a file name with the same name down the line. 
[See](https://pkg.go.dev/os#MkdirAll)